recipes-multimedia/rpidistro-ffmpeg/rpidistro-ffmpeg_4.3.2.bb

```diff
@@ -33,23 +33,31 @@ RPROVIDES:${PN} = "${PROVIDES}"
 DEPENDS = "nasm-native"
 
 inherit autotools pkgconfig
-PACKAGECONFIG ??= "avdevice avfilter avcodec avformat swresample swscale postproc avresample \
-                   opengl udev sdl2 ffplay alsa bzlib lzma pic pthreads shared theora zlib \
-                   libvorbis x264 gpl sand rpi vout-drm vout-egl \
+PACKAGECONFIG ??= "avdevice avfilter avcodec avformat swresample swscale postproc avresample ffplay \
```

Comment:
A fair bit of PACKAGECONFIG changes around. We will need to either split them in a separate commit or mention/explain them in git log.

Reply:
Yes, closing this one. The next one should hopefully have a better explanation.